.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**This is a Teamwork passion project and not officially supported by the team**
+
+The creators and maintainers of this project are doing it with enthusiasm and happiness, in odd moments between other projects and possibly even in their spare time. Please be kind to them, and understand that although they hope to get the time & energy to fix your bug :crossed_fingers:, this is not an officially supported product or service, and therefore doing so is not a part of their day jobs.
```

Comment:
feature request text mentions fixing a bug, this is tiny and no one will be confused, just nitpicking 😆

Reply:
Yes, I think we could also use the Feature Request template we use internally here. Eg:

Feature Request Title (if there are multiple request, list them in this section rather than repeating the section👍):


Short Summary / Key customer Quote (for all requests if there are multiple) go here (preferably not too long):


Explanation of use case example / potential usefulness / pain solved

Reply:
> feature request text mentions fixing a bug, this is tiny and no one will be confused, just nitpicking 😆

Whoops, I accidentally wiped out my text and copied from the bug one, forgetting that it contained the word "bug". Good catch.

Reply:
@holly-mcgill I've incorporated those items. I think that the summary one is already there in the form of the instructions under "Describe the solution".
I used the Github defaults as much as possible under the assumption that people on Github are likely to have seen them before.